src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java

```diff
@@ -127,15 +134,38 @@ public boolean isLightweight() {
         delegate.setPoll(true);
         delegate.setChangelog(true);
         try (WorkspaceList.Lease lease = computer.getWorkspaceList().acquire(dir)) {
-            delegate.checkout(build, dir, listener, node.createLauncher(listener));
```

Comment:
[diff for review](https://github.com/jenkinsci/workflow-cps-plugin/pull/147/files?w=1#diff-a2f3f524db6fdc8eb18f8f3130eae7f6R137)